appveyor.yml

```diff
@@ -230,3 +253,13 @@ deploy_script:
       py -m pip install requests
       py appveyor\mozillaSyms.py
      }
+
+on_finish:
+  # add a message to point to the NVDA build, to make testing PR's easier.
+  - ps: |
+      $appVeyorUrl = "https://ci.appveyor.com"
+      $exe = Get-ChildItem -Name output\*.exe
+      if($?){
+        $exeUrl="$appVeyorUrl/api/buildjobs/$env:APPVEYOR_JOB_ID/artifacts/output/$exe"
+        Add-AppveyorMessage "[Build (for testing PR)]($exeUrl)"

```

Comment:
As much as I get the logic, I feel this does get a bit too noisy. Especially as it will send an email to anyone associated with the pr, and at first glance looks like the build failed. I'd prefer for now not to include this final part.

Reply:
I guess what you mean is that you don't want any comment on successful builds. This is configured in the appveyor settings for the project, along with a template for how these "messages" are formatted. It is in the "notifications" section. I can take this out for now until we discuss it further, but incidentally I've noticed PR's get more testing when a link to the build is easy to access. The process of manually pasting the link in is annoying and easy to forget. 

It would be easy to adjust the template to start with a "Success!" as a heading or whatever. For now I'll disable messages on successful builds.

Reply:
For the last two messages from appveyor([success](https://github.com/nvaccess/nvda/pull/10604#issuecomment-564158254) and [failure](https://github.com/nvaccess/nvda/pull/10604#issuecomment-564146161)), the notification template looked like this:

```
{{#jobs}}
{{#messages}}
- {{message}}
{{/messages}}
{{/jobs}}


{{#failed}}
See [test results]({{buildUrl}}/tests) for [{{status}} build]({{buildUrl}}) of [commit {{commitId}}]({{commitUrl}})
{{/failed}}


{{#passed}}
See [{{status}} build]({{buildUrl}}) of [commit {{commitId}}]({{commitUrl}})
{{/passed}}
```
This is a "Mustache template" the [appveyor docs in the "customizing  message template" section](https://www.appveyor.com/docs/notifications/#customizing-message-template) have some examples, and a link to a [more formal description of the format](http://mustache.github.io/mustache.5.html). On the same page, but further up there is a description of the [available variables in the "webhook payload" section](https://www.appveyor.com/docs/notifications/#webhook-payload-default).